src/arc.ts

```diff
@@ -122,59 +124,79 @@ export async function initializeArc(provider?: any): Promise<boolean> {
   let success = false;
   let arc: any;
 
-  try {
```

Comment:
Since this method guarantees no exceptions, it should catch all.  Should restore this try/catch.  Anyone with and interest in seeing the raw details, which would usually only be a technical person, can look at the console.  If we're going to change how this function works, we need to do it thoughtfully, looking at everywhere it is used.

Reply:
fixed here:  https://github.com/daostack/alchemy/pull/1420/commits/fe770fed6f243a02c260cda42220a8d8b957e44a

Reply:
I'm not really agreeing with this at all, but we can discuss at a more opportune time :-)

Reply:
yes, refactor that if desired when we can take the time to do it right.